src/ikpy/chain.py

```diff
@@ -5,6 +5,8 @@
 """
 
 from . import URDF_utils
+from . import blender_utils
```

Comment:
I would suggest to avoid importing this directly in chain.py, but rather in the `from_blend_file` function (since there is only one function using it)